ts/features/connectivity/saga/index.ts

```diff
@@ -71,6 +71,9 @@ export function* connectionStatusSaga(
 }
 
 export default function* root(): IterableIterator<ReduxSagaEffect> {
-  const client = yield* call(createConnectivityClient, apiUrlPrefix);
+  const client = yield* call(
+    createConnectivityClient,
+    "http://192.168.0.146:3000"
```

Comment:
Testing leftovers, I guess 😄
```suggestion
   apiUrlPrefix
```

Reply:
yup, missed in development!

Reply:
addressed [156596b](https://github.com/pagopa/io-app/pull/7252/commits/156596b696deed71d76a3a7cfefee26b759cb56c)